gbpservice/neutron/nsf/configurator/api/v1/controllers/constants.py

```diff
@@ -0,0 +1 @@
+TOPIC = "configurator"
```

Comment:
If this constant is used in single location, it can be placed at the start of that file.